src/components/Filters/SchemaSieve.ts

```diff
@@ -260,29 +235,15 @@ export const getModelFilter = (
 	field: string,
 	operator: Operator<OperatorSlugs>,
 	value: any,
-	isFullTextSearch?: boolean,
-	operatorConverter?: (
-		operator: Operator<OperatorSlugs>,
-		operators: any,
-	) => Operator<OperatorSlugs>,
 ): JSONSchema => {
-	let newOperator = operator;
-	let subSchema = schema;
-	if (isFullTextSearch) {
-		subSchema = { ...schema };
-		delete subSchema.oneOf;
-		delete subSchema.enum;
-	}
+	const newOperator = operator;
+	const subSchema = schema;
```

Comment:
```suggestion
```
Atm we are just renaming these vars w/o reason. Let's just use `operator` directly and rename the `schema` argument to `subSchema`.

Reply:
Definitely a holdover from a past change. Thank you :)